pkgs/tools/audio/pasystray/default.nix

```diff
@@ -17,12 +17,17 @@ stdenv.mkDerivation rec {
   patches = [
     # https://github.com/christophgysin/pasystray/issues/90#issuecomment-306190701
     ./fix-wayland.patch
+
+    (fetchpatch {
```

Comment:
please comment that it allows us to replace `libappindicator` with `libayanata-appindicator`

Reply:
Done!